index.md

```diff
@@ -58,104 +58,105 @@ This page introduces guidelines and other standards related to web accessibility
 
-* natural information such as text, images, and sounds
-* code or markup that defines structure, presentation, etc.
+Las herramientas de creación de contenido son programas o servicios que los “autores” (desarrolladores web, diseñadores, redactores, etc) utilizan para producir contenido web. Por ejemplo: editores de HTML, sistemas de gestión de contenidos (CMS) y sitios web que permiten a los usuarios añadir contenido tales como blogs y redes sociales. Los documentos ATAG explican cómo:
 
-WCAG applies to dynamic content, multimedia, "mobile", etc. WCAG can also be applied to non-web information and communications technologies (ICT), as described in [WCAG2ICT](/standards-guidelines/wcag/non-web-ict/).
+* hacer que las herramientas de creación de contenido sean accesibles en sí mismas, de forma que las personas con discapacidad puedan crear contenido web, y
+* ayudar a los autores a crear contenido web más accesible.
 
-WCAG info:
-- [WCAG Overview](/standards-guidelines/wcag/)
-- [[WCAG 2.1 at a Glance]](/standards-guidelines/wcag/glance/)
-- [How to Meet WCAG 2 (Quick Reference)](https://www.w3.org/WAI/WCAG21/quickref/)
-- [WCAG 2.0 Standard](https://www.w3.org/TR/WCAG20/)
-- [WCAG 2.1 Standard](https://www.w3.org/TR/WCAG21/)
+Información sobre las pautas ATAG:
+- [Resumen de ATAG](/standards-guidelines/atag/)
+- [[ATAG de un vistazo]](/standards-guidelines/atag/glance/)
+- [Estándar ATAG 2.0](https://www.w3.org/TR/ATAG/)
 
-### Authoring Tool Accessibility Guidelines (ATAG) {#atag}
+### Pautas de accesibilidad para el agente de usuario (UAAG) {#uaag}
 
-Authoring tools are software and services that "authors" (web developers, designers, writers, etc.) use to produce web content. For example: HTML editors, content management systems (CMS), and websites that let users add content, such as blogs and social networking sites. ATAG documents explain how to:
-* make the authoring tools themselves accessible, so that people with disabilities can create web content, and
-* help authors create more accessible web content.
+Los agentes de usuario son navegadores, extensiones de los navegadores, reproductores multimedia, lectores y otras aplicaciones que presentan contenido web.
 
-ATAG info:
-- [ATAG Overview](/standards-guidelines/atag/)
-- [[ATAG at a Glance]](/standards-guidelines/atag/glance/)
-- [ATAG 2.0 Standard](https://www.w3.org/TR/ATAG/)
+Información sobre las pautas UAAG:
+- [Resumen de UAAG](/standards-guidelines/uaag/)
+- [Nota sobre UAAG 2.0](https://www.w3.org/TR/UAAG20/)
 
-### User Agent Accessibility Guidelines (UAAG) {#uaag}
-
-User agents include browsers, browser extensions, media players, readers, and other applications that render web content.
-
-UAAG info:
-- [UAAG Overview](/standards-guidelines/uaag/)
-- [UAAG 2.0 Note](https://www.w3.org/TR/UAAG20/)
-
-## Technical Specifications
+## Especificaciones técnicas
 
 ### Accessible Rich Internet Applications (WAI-ARIA) {#aria}
```

Comment:
This would be a case similar to the one I commented above. In this case, the words are kept in English but I think they should be translated. This also brings what I would say that are some inconsistencies in the text since there are cases in which they are translated (like in the line 101 the text Pautas de accesibilidad para el agente de usuario (UAAG)) and other cases in which are not like this one.